conf/application.conf.template

```diff
@@ -29,9 +29,8 @@ play {
     evolutions.autocommit              =   false
     evolutions.db.default.autoApply    =   true
     ws.timeout.connection              =   10000ms
-    filters.headers.contentSecurityPolicy = "default-src 'self'; script-src 'self' '%NONCE-SOURCE%' https://forums.spongepowered.org https://forums-cdn.spongepowered.org https://www.google-analytics.com/analytics.js; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com https://forums-cdn.spongepowered.org; img-src 'self' https:; font-src 'self' https://fonts.gstatic.com; frame-src https://forums.spongepowered.org; frame-ancestors 'none'; block-all-mixed-content"
-    filters.headers.contentSecurityPolicy = ${?CONTENT_SECURITY_POLICY}
-    filters.enabled                   += "filters.MimeKeyedCspFilter"
+    filters.headers.contentSecurityPolicy = null
+    filters.enabled                    =  ${play.filters.enabled} ["security.NonceFilter", "filters.MimeKeyedCspFilter"]
```

Comment:
I think you forgot to add the CSRFFilter here.

Reply:
This is now enabled by default in Play 2.6: https://www.playframework.com/documentation/2.6.x/Filters#default-filters

Reply:
Ohh good catch! My bad then :) Will check if this stuff is working when I get some time